src/dune_util/dune

```diff
@@ -1,3 +1,3 @@
 (library
  (name dune_util)
- (libraries stdune xdg dune_lang memo))
+ (libraries stdune xdg build_path_prefix_map dune_lang memo))
```

Comment:
Do we need to remove this dependency from Dune engine?

Reply:
Good question. I tried it but it wasn't possible. Dune_util still re-exports the `Build_path_prefix_map.map` type in the api, so we nee to depend on `build_path_prefix_map` to see the implementation of this type.